search/tests/api/extensions/field.spec.js

```diff
@@ -1,7 +1,7 @@
 /**
  * @jest-environment node
  */
-describe('STAC API Extensions', () => {
+describe.skip('STAC API Extensions', () => {
```

Comment:
Why are we skipping a test? Mistake?

Reply:
@abbottry The tests were not functional and are functionally useless. They were checking if `true` was truthy. Fixing the tests to check what they were for would be outside the scope of the ticket.

Reply:
Why did it ever make it to the repo?